### PR TITLE
Html-css mobile view pull request

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
Add html file for mobile view website
Add css to style html elements
Add image file for images

### CODE REVIEW

**Comment by Kaweesi-Matia** [comment]
Add html file for the mobile view website
Add css file for styling html elements
Add images

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+</head>
+<body>
+  <div class="wrapper">
+    <div class="wrapper-1">
+      <header>
+        <div class="logo"><a href="">My Logo</a></div>
+        <nav>
+          <a href="">
+            <img id="menu" src="./images/menu.png" alt="menu">
+          </a>
```
Comment (thecodechaser) [inline index.html]: - [ ] You did an excellent job on the design but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template as much as possible. Please reduce the size of the hamburger button to match the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+    <section class="content">
+      <h1>I’m Kaweesi  Glad to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a id="connect" href="">LET’S CONNECT</a>
+      <ul class="content-list">
+        <li>
+          <a href=""><img src="./images/twitter.png" alt="twitter"></a>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please reduce this white space between the header and hero section and make sure your design matches with the Figma design.
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <img id="menu" src="./images/menu.png" alt="menu">
+          </a>
+
+        </nav>
+
+      </header>
+      
+    </div>
+    <section class="content">
+      <h1>I’m Kaweesi  Glad to see you!</h1>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please increase the font weight for the main heading and make sure it matches the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Kaweesi-Matia👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -15,7 +15,7 @@ h1 {
   margin-left: 20px;
   color: #172b4d;
   font-style: normal;
-  font-weight: bolder;
+  font-weight: 900px;
```
Comment (leonardodiasb) [inline style.css]: - [ ] Font-weights do not accept `px`. Please take a look at [this](https://www.w3schools.com/cssref/pr_font_weight.asp) article and fix your attribute value.(setting `font-weight: bolder;` was correct)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  max-width: 40%;
+  position: relative;
+  left: -15px;
+  padding-right: 10px;
+}
+
+.wrapper-1 {
+  height: 80px;
+  background-color: white;
+  margin-bottom: 100px;
```
Comment (leonardodiasb) [inline style.css]: - [ ] As the previous code reviewer mentioned, kindly reduce the whitespace to match with the Figma design.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Kaweesi-Matia, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

No linter errors. ✔️
GitHub flow is followed. ✔️
Professional README file. ✔️
PR has a good title and summary. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Kindly use the Figma template font family to make the style of your app look the same as the template. This was noticed by the previous code reviewer when he mentioned that the font weight was off. Actually, the font families were making the difference. Try applying the `Poppins` font family and remember to import different font weights and see how your app will look.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
### **Status: Approved**

Hi @Kaweesi-Matia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark:
-  GitHub flow is followed. :heavy_check_mark:
-  Professional README file. :heavy_check_mark:
-  PR has a good title and summary. :heavy_check_mark: 



- [OPTIONAL] Kindly reduce the space between your header and the paragraph. So as to make it looks almost the same like the provided Figma design

space

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**